src/Workspaces/Core/Portable/Remote/RemoteFeatureOptions.cs

```diff
@@ -15,15 +14,30 @@ internal static class RemoteFeatureOptions
         /// Global switch that determines of OOP can be used for a language feature. Exposed through
         /// user visible switch to let people opt-in/out of this behavior.
         /// </summary>
-        public static readonly Option<bool> OutOfProcessAllowed = new Option<bool>(
+        public static readonly Option<bool> ExperimentalOutOfProcessAllowed = new Option<bool>(
             nameof(RemoteFeatureOptions), nameof(OutOfProcessAllowed), defaultValue: false,
             storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(OutOfProcessAllowed)));
 
+        /// <summary>
+        /// This field contains the original name of the experimental OOP feature, which is used as the key for saving
```

Comment:
The pattern we have here is just to stop using nameof. TBH I consider using nameof to be an anti-pattern in synced options in general precisely for this reason...